### DISCUSSION TITLE
Tasmota Web Installer with NodeMCU v2 (Amica) hangs

### DISCUSSION THREAD

**Discussion by alexwala** [arendst/Tasmota#15170] [Q&A] [answered]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
Trying to flash a NodeMCUv2 board (amica, cp2102) via tasmota web installer.
Hangs on 'Preparing Installation'.
Using Tasmotizer by checking 'Self-resetting device (NodeMCU, Wemos)' works fine.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): NodeMCU v2 (Amica)
- [x] Tasmota binary firmware version number used: v11.0.0
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: Tasmota Web Installer, Tasmotizer
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
Try flash NodeMCU v2 (Amica) via Tasmota Web Installer

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
Flashing is performed.

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment, marked as answer]
Does a other NodeMCU or WemosD1 mini works with WebInstaller?

**Comment by alexwala** [comment]
Actually having only one: NodeMCU v2 (Amica)

**Reply by barbudor** [reply]
Are you sure that Tasmotizer or any other software is not using the COM port while trying to flash from Web flasher ?

You may also close all browser windows, check with the appropriate task manager of your OS that no hidden browser is running and try again with only 1 browser window/tab open.

**Comment by alexwala** [comment]
>>> Are you sure that Tasmotizer or any other software is not using the COM port while trying to flash from Web flasher ?
Yes

>>> You may also close all browser windows, check with the appropriate task manager of your OS that no hidden browser is running and try again with only 1 browser window/tab open.
done, rebooted, ecc - no success

After hitting 'Install', blue led on esp module blinks one time, then a little break, an than four times -than nothing happens.

web installer hangs on:
<img width="376" alt="tasmota" src="https://user-images.githubusercontent.com/70528897/159219088-215d0e7b-5fba-4757-b8fe-50c3851800a5.PNG">

**Comment by Jason2866** [comment]
Known problem https://github.com/esphome/esp-web-tools/issues/156
